README.md

```diff
@@ -153,11 +153,14 @@ Follow these [instructions](https://hatch.pypa.io/latest/install/) to install it
 ### Tests
 
 ```sh
-# Run all tests
-hatch run test
+# Run all tests for all Python versions.
+hatch test --all
```

Comment:
Does `hatch test` work and pick a default Python version?

Reply:
You mean the default python version? I assume it's using current python version which installed hatch.

Reply:
I would leave it documented here. This is likely the command most people will use.

Reply:
Good point. Done.